contents/blog/best-mobile-app-ab-testing-tools.mdx

```diff
@@ -0,0 +1,303 @@
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/open-source-testing-tools/testinghog.png
+featuredImageType: full
+author:
+  - lior-neu-ner
+category: General
+tags:
+  - Guides
+  - Comparisons
+---
+
+import { ComparisonTable } from 'components/ComparisonTable'
+import { ComparisonRow } from 'components/ComparisonTable/row'
+
+Also known as split testing or [multivariate testing](/product-engineers/what-is-multivariate-testing-examples), A/B testing enables you to measure the impact of product changes on your mobile apps.
+
+There are many A/B testing tools out there, but each one is built for a specific target audience in mind. In this guide, we’ll explore these options so that you can decide which is best for you.
+
+## 1. PostHog
+
+![PostHog - best ab testing tools](../images/blog/best-mobile-app-ab-testing-tools/posthog.png)
+
+- **Best for:** Engineering and product teams
+- **Free tier:** <span className="text-green text-lg">✔</span>
+- **Transparent pricing:** <span className="text-green text-lg">✔</span>
+- **Open source:** <span className="text-green text-lg">✔</span>
+- **Get started without talking to sales:** <span className="text-green text-lg">✔</span>
+- **Includes product analytics:** <span className="text-green text-lg">✔</span>
+
+[PostHog](https://posthog.com) (that's us!) is an all-in-one platform built to help engineers create better products. It includes A/B testing **and** a whole bunch more, such as [product analytics](/product-analytics), [session replays](/session-replay), [feature flags](/feature-flags), and [surveys](/survey).
+
+It's designed for product-minded engineers, growth teams, and product managers who need to move fast and iterate based on reliable, actionable insights.
+
+### Which platforms does PostHog support?
+
+<ComparisonTable column1="PostHog">
+  <ComparisonRow column1={true} feature="Android" description="" />
+  <ComparisonRow column1={true} feature="iOS" description="" />
+  <ComparisonRow column1={true} feature="React Native" description="" />
+  <ComparisonRow column1="Product analytics and feature flags only" feature="Flutter" description="" />
+  <ComparisonRow column1={true} feature="Web" description="" />
+  <ComparisonRow column1={true} feature="Server" description="" />
+</ComparisonTable>
+
+### How much does PostHog cost?
+
+PostHog has [transparent pricing]((/pricing)) based on the usage. It’s free to get started and completely free for the first 1 million events. After this free monthly allowance, you'll pay $0.0001/event, and PostHog charges progressively less the more you use. You can also set billing limits to ensure you don't get surprise bills. 
+
+While PostHog is open source, its A/B testing features aren't included in the open source release.
+
+> **Further reading:** New to A/B testing? Read [A software engineer's guide to A/B testing](/blog/ab-testing-guide-for-engineers) and our guide to [common A/B testing mistakes](/blog/ab-testing-mistakes).
```

Comment:
```suggestion
> **Further reading:** New to A/B testing? Read [a software engineer's guide to A/B testing](/blog/ab-testing-guide-for-engineers) and our guide to [common A/B testing mistakes](/blog/ab-testing-mistakes).
```
Even though this is a title, it looks a bit weird. Should be in quotes if you want to match the formatting.